### PR TITLE
Antifam Download page + TS

### PR DESCRIPTION
**Pull request by gustavo-salazar**
* Creates the download page for AntiFam data.
* Migrates to TS + VF2:
  * The components in the Download page
  * The Tabs component

### CODE REVIEW

File: src/components/DownloadFTP/AntiFam/index.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+import cssBinder from 'styles/cssBinder';
+
+import ipro from 'styles/interpro-vf.css';
+import fonts from 'EBI-Icon-fonts/fonts.css';
+
+const css = cssBinder(fonts, ipro);
+
+const AntiFamDownloads = () => (
+  <>
```
[the remark this answers is not in the document]
Comment (gustavo-salazar) [reply src/components/DownloadFTP/AntiFam/index.tsx:13]: Yes, it is. The `<>` tag is a react artefact to be able to group inner elements but without rendering anything in the final html

File: src/components/DownloadFTP/AntiFam/index.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>
+            <Link href="https://ftp.ebi.ac.uk/pub/databases/Pfam/AntiFam/current/Antifam.tar.gz">
+              AntiFam models
+            </Link>
+          </td>
+          <td>HMMs for AntiFam</td>
```
Comment (matthiasblum) [inline src/components/DownloadFTP/AntiFam/index.tsx:31]:
```suggestion
          <td>Seed alignments and profile-HMMs</td>
```

File: src/components/DownloadFTP/AntiFam/index.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+          <th className={css('xs-hide')}>File name</th>
+          <th className={css('xs-hide')}>Format</th>
+          <th className={css('xs-hide')}>Links</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>
+            <Link href="https://ftp.ebi.ac.uk/pub/databases/Pfam/AntiFam/current/Antifam.tar.gz">
+              AntiFam models
```
Comment (matthiasblum) [inline src/components/DownloadFTP/AntiFam/index.tsx:28]:
```suggestion
            <Link href="https://ftp.ebi.ac.uk/pub/databases/Pfam/AntiFam/current/Antifam.tar.gz">
              AntiFam families
```

File: src/components/DownloadFTP/AntiFam/index.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+              />
+            </Link>
+            <br />
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <p className={css('small', 'margin-top-small')}>
+      See all downloads available on the{' '}
+      <Link href="https://ftp.ebi.ac.uk/pub/databases/Pfam/AntiFam/current/">
```
Comment (matthiasblum) [inline src/components/DownloadFTP/AntiFam/index.tsx:48]:
```suggestion
      <Link href="https://ftp.ebi.ac.uk/pub/databases/Pfam/AntiFam/">
```

**Review by matthiasblum — APPROVED** [review]
LGTM. Made a few minor comments.

Also, _PRINTS_ should be in upper case in the title of the tab:

https://github.com/ProteinsWebTeam/interpro7-client/blob/a366e1250371597a4d9a52eb5e547f0e7f3d6550/src/pages/Download/index.tsx#L74

All tabs, except the one for InterPro, have less than five downloads. Not the scope of this PR but I am wondering if it wouldn't be better/simpler to display all downloads in one single page.

**Pull request merged**